.werft/jobs/build/job-config.ts

```diff
@@ -0,0 +1,161 @@
+import { config } from "process";
+import { exec } from "../../util/shell";
+import { Werft } from "../../util/werft";
+
+export interface JobConfig {
+    analytics: string
+    buildConfig: any;
+    cleanSlateDeployment: boolean
+    coverageOutput: string
+    dontTest: boolean;
+    dynamicCPULimits: boolean;
+    installEELicense: boolean
+    localAppVersion: string
+    mainBuild: any;
```

Comment:
```suggestion
    mainBuild: boolean;
```

Could that be done? Is there any place where we don't use this as a boolean?

Reply:
I can give it a go - right now the types are based on what TypeScript interpreted the types to be ☺️